drivers/hmis/app/graphql/mutations/delete_enrollment.rb

```diff
@@ -13,12 +13,12 @@ def resolve(id:)
         if enrollment.in_progress?
           enrollment.destroy
         else
-          errors << InputValidationError.new('Only in-progress enrollments can be deleted')
+          errors << HmisErrors::CustomValidationError.new(:base, full_message: 'Completed enrollments can not be deleted. Please exit the client instead.')
```

Comment:
This may need to happen, is there a way to override this? If someone entered an enrollment for the wrong client and went all the way through the intake assessment, they may need to still be able to delete the enrollment, but definitely with a confirmation process.

Reply:
created https://www.pivotaltracker.com/story/show/184402680